docs/integrate/testing.md

```diff
@@ -0,0 +1,56 @@
+(testing)=
+# Software Testing with CrateDB
+
+This documentation section lists Java- and Python-based test frameworks
+and libraries that support software integration testing with CrateDB.
+
+
+(python-pytest)=
+## Python pytest
+
+The popular [pytest] framework makes it easy to write small tests, yet
+scales to support complex functional testing for applications and libraries.
+The [pytest-crate] package manages CrateDB instances for running integration
+tests against them. It is based on [cr8](#cr8), and provides the `crate`,
+`crate_execute`, and `crate_cursor` pytest fixtures on top.
```

Comment:
sorry, this `on top` is a bit confusing, on top of what? or do you mean crate_execute and crate_cursor on top of crate?

Reply:
Rephrasing to use:

> It is based on [cr8](#cr8) for the heavy lifting, and additionally provides
the `crate`, `crate_execute`, and `crate_cursor` pytest fixtures for
developer convenience.